src/EnergyPlus/DXCoils.cc

```diff
@@ -1230,13 +1231,17 @@ namespace DXCoils {
 				}
 			}
 
-			DXCoil( DXCoilNum ).Twet_Rated( 1 ) = Numbers( 6 );
-			DXCoil( DXCoilNum ).Gamma_Rated( 1 ) = Numbers( 7 );
-			DXCoil( DXCoilNum ).MaxONOFFCyclesperHour( 1 ) = Numbers( 8 );
-			DXCoil( DXCoilNum ).LatentCapacityTimeConstant( 1 ) = Numbers( 9 );
+			//Set minimum OAT for compressor operation
+			DXCoil( DXCoilNum ).MinOATCompressor = Numbers( 6 );
+			if ( NumNumbers < 6 ) DXCoil( DXCoilNum ).MinOATCompressor = minOATCompDXCooling; // input field is after min fields and won't default if field not included 
```

Comment:
Yeah, this one is always tricky.  In my mind, I think of `\min-fields` as "the minimum number of fields that must be interpreted - blank or otherwise - by the simulation model in order to perform a simulation".  If you are saying you need a default to be inserted there by forcing it to be blank in order to make a good decision on your modeling, then I say it should be in the min fields.  Not sure this is the right time to have such a philosophical discussion about min fields, so for this PR just make it sufficient for a robust calculation with your new code and carry on.  Maybe we'll have those philosophical discussions over a hackathon sometime...

All that said, let me know what changes you are planning to make here and I'll wrap up my review when it is all ready to go.